mycpp/leaky_mylib.cc

```diff
@@ -242,7 +242,9 @@ void FormatStringer::format_s(Str* s) {
 }
 
 void FormatStringer::format_o(int i) {
-  NotImplemented();
+  data_ = static_cast<char*>(realloc(data_, len_ + kIntBufSize));
```

Comment:
`kIntBufSize` might be wrong size?

Reply:
I think this is fine, the comments around kIntBufSize seem to indicate it

(although they are giving me a headache now, probably should change it to 11 or 12 :) )